iothub/service/src/Feedback/Models/FeedbackRecord.cs

```diff
@@ -8,67 +8,53 @@ namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// Contains properties of an individual feedback record.
-    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
     /// </summary>
+    /// <remarks>
+    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
+    /// </remarks>
     public class FeedbackRecord
     {
         /// <summary>
         /// The Id of the cloud-to-device message to which this feedback message relates.
-        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
         /// </summary>
+        /// <remarks>
+        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
+        /// </remarks>
         [JsonProperty(PropertyName = "originalMessageId", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string OriginalMessageId { get; set; }
+        public string OriginalMessageId { get; internal set; }
 
         /// <summary>
         /// The device generation Id of the target device of the cloud-to-device message to which this piece of feedback relates.
+        /// </summary>
+        /// <remarks>
         /// DeviceGenerationId is an IoT hub generated, case-sensitive string. This value is used to distinguish devices with the
         /// same device Id when they have been deleted and re-created.
-        /// </summary>
+        /// </remarks>
         [JsonProperty(PropertyName = "deviceGenerationId", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string DeviceGenerationId { get; set; }
+        public string DeviceGenerationId { get; internal set; }
 
         /// <summary>
         /// The Id of the target device of the cloud-to-device message to which this piece of feedback relates.
         /// </summary>
         [JsonProperty(PropertyName = "deviceId", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string DeviceId { get; set; }
+        public string DeviceId { get; internal set; }
 
         /// <summary>
         /// Date and time that indicates when the outcome of the message happened in UTC (for example, the hub received the feedback message or the original message expired).
         /// </summary>
         [JsonProperty(PropertyName = "enqueuedTimeUtc", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public DateTime EnqueuedTimeUtc { get; set; }
+        public DateTime EnqueuedTimeUtc { get; internal set; }
 
         /// <summary>
         /// The status of feedback messages that are generated by the IoT hub.
         /// </summary>
-        /// <remarks>
-        /// Possible values are:
-        /// <list type="bullet">
```

Comment:
Don't document possible values for enum that is part of the signature and will have a link to the possible values.